src/xdsl/dialects/builtin.py

```diff
@@ -51,8 +58,14 @@ class SymbolNameAttr(ParametrizedAttribute):
     data = ParameterDef(StringAttr)
 
     @staticmethod
-    def get(data: str) -> SymbolNameAttr:
-        return SymbolNameAttr([StringAttr.get(data)])
+    @builder
```

Comment:
All builders have to be static, right. Maybe we could attach `@staticmethod` inside the `@builder` decorator, but this might be a bad idea.

Reply:
The problem here is that pycharm complains if you do not include the `@staticmethod`, since it doesn`t understand that the function is static.